src/site/_data/site.js

```diff
@@ -31,4 +31,16 @@ module.exports = {
   // of the site. Otherwise all image paths are local. This means you can
   // develop locally without having to mess with the CDN at all.
   imageCdn: "https://webdev.imgix.net",
+  analytics: {
+    ids: {
+      prod: "UA-126406676-2",
+      staging: "UA-126406676-3",
```

Comment:
We don't use non-prod at all right now? That's intentional?

Reply:
Hm good point. I guess I can toggle which analytics to use based on the ELEVENTY_ENV